<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PortScan</class>
 <widget class="QWidget" name="PortScan">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>563</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>端口扫描</string>
  </property>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>541</width>
     <height>281</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>华文中宋</family>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="toolTip">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt; font-weight:600;&quot;&gt;格式为:IP/协议/端口&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:11pt; font-weight:600;&quot;&gt;【Sample】&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:11pt;&quot;&gt;1. 192.168.10.100/TCP/80 #扫描该IP的TCP协议的80端口&lt;/span&gt;&lt;/p&gt;&lt;p&gt;&lt;span style=&quot; font-size:11pt;&quot;&gt;2. 192.168.10.100/UDP/53 #扫描该IP的UDP协议的53端口&lt;/span&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
   <property name="styleSheet">
    <string notr="true">border-radius:5px;
background-color: lightgray;</string>
   </property>
   <property name="title">
    <string>端口扫描</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <widget class="QLabel" name="labelInput">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>30</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>华文中宋</family>
      <pointsize>11</pointsize>
      <weight>50</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 11pt &quot;华文中宋&quot;;</string>
    </property>
    <property name="text">
     <string>输入扫检测址信息:</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditInput">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>30</y>
      <width>271</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:5px;
background-color: rgb(255, 255, 255);
font: 10pt &quot;Courier&quot;;</string>
    </property>
    <property name="placeholderText">
     <string>请输入待扫描IP地址</string>
    </property>
    <property name="clearButtonEnabled">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonStart">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>30</y>
      <width>81</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>11</pointsize>
      <weight>9</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(85, 85, 127);
border-radius:5px;
font: 75 11pt &quot;微软雅黑&quot;;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>点击检查</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="textEditScanResult">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>110</y>
      <width>501</width>
      <height>161</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(255, 255, 255);
border-radius:5px;</string>
    </property>
    <property name="placeholderText">
     <string>结果信息输出</string>
    </property>
   </widget>
   <widget class="QLabel" name="labelScanIP">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 11pt &quot;华文中宋&quot;;</string>
    </property>
    <property name="text">
     <string>输入扫描地址信息：</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEditScanIP">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>70</y>
      <width>271</width>
      <height>20</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border-radius:5px;
background-color: rgb(255, 255, 255);
font: 10pt &quot;Courier&quot;;</string>
    </property>
    <property name="placeholderText">
     <string>请输入IP地址,如192.168.10.100</string>
    </property>
    <property name="clearButtonEnabled">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButtonScanIP">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>70</y>
      <width>81</width>
      <height>23</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>微软雅黑</family>
      <pointsize>11</pointsize>
      <weight>9</weight>
      <italic>false</italic>
      <bold>false</bold>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(85, 85, 127);
font: 75 11pt &quot;微软雅黑&quot;;
border-radius:5px;
color: rgb(255, 255, 255);</string>
    </property>
    <property name="text">
     <string>点击扫描</string>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
